### PR TITLE
FIX: parameter refactoring: fix intelmqdump

### LINKED ISSUE #1782: "intelmqctl list queues" raises exception (parameter refactoring)
```
> intelmqctl list queues -q
Traceback (most recent call last):
  File "/usr/local/bin/intelmqctl", line 11, in <module>
    load_entry_point('intelmq', 'console_scripts', 'intelmqctl')()
  File "/home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py", line 1899, in main
    return x.run()
  File "/home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py", line 1041, in run
    retval, results = args.func(**args_dict)
  File "/home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py", line 1224, in list
    return self.list_queues(non_zero=non_zero, count=count)
  File "/home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py", line 1290, in list_queues
    pipeline = PipelineFactory.create(self.parameters, logger=self.logger)
TypeError: create() got multiple values for argument 'logger'
```

### LINKED ISSUE #1822: intelmqdump recovery fails on develop
```
> intelmqdump ...
...
(r)ecover by ids, recover (a)ll, delete (e)ntries, (d)elete file, (s)how by ids, (q)uit, edit id (v)? a
Traceback (most recent call last):
  File "/usr/local/bin/intelmqdump", line 11, in <module>
    load_entry_point('intelmq', 'console_scripts', 'intelmqdump')()
  File "/home/sebastianw/intelmq/intelmq/bin/intelmqdump.py", line 333, in main
    pipe = pipeline.PipelineFactory.create(params, logger)
  File "/home/sebastianw/intelmq/intelmq/lib/pipeline.py", line 48, in create
    broker = broker.title()
AttributeError: 'Logger' object has no attribute 'title'
```
Probably related to parameter refactoring

### PR DESCRIPTION
**Pull request by waldbauer-certat**
Closes #1782 
Closes #1822

### CODE REVIEW

**Review by wagner-certat — DISMISSED** [review]
Works for me.

Can you please add a changelog entry and add the issue number to the commit message as well?

File: intelmq/bin/intelmqdump.py [hunk 1]
```diff
@@ -327,10 +328,10 @@ def main():
                 save_file(handle, content)
             elif answer[0] == 'r':
                 # recover entries
-                default = utils.load_configuration(DEFAULTS_CONF_FILE)
+                #default = utils.load_configuration(DEFAULTS_CONF_FILE)
                 runtime = utils.load_configuration(RUNTIME_CONF_FILE)
-                params = utils.load_parameters(default, runtime)
-                pipe = pipeline.PipelineFactory.create(params, logger)
+                #params = utils.load_parameters(default, runtime)
+                pipe = pipeline.PipelineFactory.create(logger=logger)
```
Comment (wagner-certat) [inline intelmq/bin/intelmqdump.py:334]: What about (untested):
```python
params = utils.load_configuration(DEFAULTS_CONF_FILE)
params.update(utils.load_configuration(RUNTIME_CONF_FILE)[botid].get('parameters', {}))
pipe = pipeline.PipelineFactory.create(parameters=params, logger=logger)
```

**Comment by wagner-certat** [comment]
Test is failing with:
```
Starting postgres service container
  /usr/bin/docker pull postgres
  Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
  Using default tag: latest
  Warning: Docker pull failed with exit code 1, back off 9.549 seconds before retry.
  /usr/bin/docker pull postgres
  Using default tag: latest
  Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
  Warning: Docker pull failed with exit code 1, back off 4.233 seconds before retry.
  /usr/bin/docker pull postgres
  Using default tag: latest
  Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
  Error: Docker pull failed with exit code 1
```

**Comment by schacht-certat** [comment]
> Works for me.
> 
> Can you please add a changelog entry and add the issue number to the commit message as well?

Do we really have to mention fixes, that fix bugs we introduces in develop, in the changelog?

**Comment by schacht-certat** [comment]
> Test is failing with:
> 
> ```
> Starting postgres service container
>   /usr/bin/docker pull postgres
>   Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
>   Using default tag: latest
>   Warning: Docker pull failed with exit code 1, back off 9.549 seconds before retry.
>   /usr/bin/docker pull postgres
>   Using default tag: latest
>   Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
>   Warning: Docker pull failed with exit code 1, back off 4.233 seconds before retry.
>   /usr/bin/docker pull postgres
>   Using default tag: latest
>   Error response from daemon: received unexpected HTTP status: 503 Service Unavailable
>   Error: Docker pull failed with exit code 1
> ```

This was a temporary error

**Pull request merged**